uamqp/management_link.py

```diff
@@ -137,16 +143,30 @@ def _on_message_received(self, message):
             )
             self._pending_operations.remove(to_remove_operation)
 
-    def _on_send_complete(self, message, reason, state):
-        if SEND_DISPOSITION_REJECT in state:  # either rejected or accepted
+    def _on_send_complete(self, message_delivery, reason, state):  # todo: reason is never used, should check spec
+        if SEND_DISPOSITION_REJECT in state:
             to_remove_operation = None
             for operation in self._pending_operations:
-                if message == operation.message:
+                if message_delivery.message == operation.message:
                     to_remove_operation = operation
                     break
             self._pending_operations.remove(to_remove_operation)
-            to_remove_operation.on_execute_operation_complete(
-                ManagementExecuteOperationResult.ERROR, None, None, message)
+            if SEND_DISPOSITION_REJECT in state:  # either rejected or accepted
```

Comment:
We already have this `if` statement on line 147 - so wont it always be true?

Reply:
nice catch! it's a mistake :P